Microsoft.Azure.CosmosRepository/src/DefaultRepository.cs

```diff
@@ -35,8 +38,11 @@ internal class DefaultRepository<TItem> : IRepository<TItem> where TItem : IItem
         public DefaultRepository(
             IOptionsMonitor<RepositoryOptions> optionsMonitor,
             ICosmosContainerProvider<TItem> containerProvider,
-            ILogger<DefaultRepository<TItem>> logger) =>
-            (_optionsMonitor, _containerProvider, _logger) = (optionsMonitor, containerProvider, logger);
+            ILogger<DefaultRepository<TItem>> logger,
+            ICosmosQueryableProcessor cosmosQueryableProcessor,
+            IRepositoryExpressionProvider repositoryExpressionProvider) =>
+            (_optionsMonitor, _containerProvider, _logger, _cosmosQueryableProcessor, _repositoryExpressionProvider) =
+            (optionsMonitor, containerProvider, logger, cosmosQueryableProcessor, repositoryExpressionProvider);

```

Comment:
```suggestion
                (optionsMonitor, containerProvider, logger, cosmosQueryableProcessor, repositoryExpressionProvider);
```

An assignment should be spaced on the next new line